remotecv/utils.py

```diff
@@ -10,10 +10,60 @@
 
 import logging
 
+from redis import Redis, Sentinel
```

Comment:
`Sentinel` is available only from `redis-py` [version `4.0.0` onwards](https://github.com/redis/redis-py/blob/master/CHANGES). Please make sure to update — cough — Poetry's dependency thing, however that is.

Reply:
I've just reviewed #43, which solves this. 👍